lib/cc/services/github_pull_requests.rb

```diff
@@ -13,13 +13,35 @@ class Config < CC::Service::Config
       label: "Github Context",
       description: "The integration name next to the pull request status",
       default: "codeclimate"
+    attribute :welcome_comment_enabled, Axiom::Types::Boolean,
+      label: "Welcome comment enabled?",
+      description: "Should Code Climate post a welcome comment on pull requests?",
+      default: false
+    attribute :welcome_comment_markdown, Axiom::Types::String,
+      label: "Welcome comment markdown",
+      description: "The body of the welcome comment for first-time contributors to this repo.",
+      default: <<-COMMENT
+* This repository is using Code Climate to automatically check for code quality issues.
+* You can see results for this analysis in the PR status below.
+* You can install [the Code Climate browser extension](https://codeclimate.com/browser) to see analysis without leaving GitHub.
+
+Thanks for your contribution!
+    COMMENT
 
     validates :oauth_token, presence: true
   end
 
   self.title = "GitHub Pull Requests"
   self.description = "Update pull requests on GitHub"
 
+  def receive_pull_request_welcome_comment
```

Comment:
Since this event is only for PR integrations, do we need `pull_request` as part of the event name? Seems redundant.